test/System/Process/TypedSpec.hs

```diff
@@ -1,30 +1,30 @@
-{-# LANGUAGE CPP #-}
+{-# LANGUAGE CPP               #-}
 {-# LANGUAGE OverloadedStrings #-}
 module System.Process.TypedSpec (spec) where
 
-import System.Process.Typed
-import System.Process.Typed.Internal
-import System.IO
-import Control.Concurrent.Async (Concurrently (..))
-import Control.Concurrent.STM (atomically)
-import Test.Hspec
-import System.Exit
-import System.IO.Temp
+import           Control.Concurrent.Async (Concurrently (..))
```

Comment:
Can you back out the whitespace changes here too?

Reply:
Sure. Forgot this change.